packages/babylon/test/fixtures/core/categorized/regex-after-function-expression/actual.js

```diff
@@ -0,0 +1 @@
+const x = function foo() {} /42/i
```

Comment:
Probably should also test `(function foo() {} /42/i)` to be sure

Reply:
And also this, since the significant line break must be considered:
```
return
function fn() {}
/42/i
```